src/utils/crud.js

```diff
@@ -58,7 +82,8 @@ const crudControllers = model => {
     getOne: getOne(model),
     createOne: createOne(model),
     updateOne: updateOne(model),
-    deleteOne: deleteOne(model)
+    deleteOne: deleteOne(model),
+    upload,
   };
 };
 
```

Comment:
Maybe we can run through this at the meeting tonight, I'm not familiar with sending files so could learn something from you!